demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -2145,3 +2148,70 @@ def test_IsHaveUnitTestFileValidator_obtain_invalid_content_items__invalid_item(
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+def test_is_valid_context_path_depth_command():
+    with ChangeCWD(REPO.path):
```

Comment:
Let's add a test case with multiple malformed outputs for the same command & an integration with multiple malformed commands.
I want to make sure the error message is readable.

Reply:
Added tests like you asked, here is an example:

The level of depth for context output path for command: ip In the yml should be less or equal to 5 check the following outputs:
path_3.3.4.5.6.7.8.9
path_1.2.3.4.5.6
The level of depth for context output path for command: file In the yml should be less or equal to 5 check the following outputs:
path_12.13.14.15.16.17.18.19
path_10.11.12.13.14.15